Sources/ITwinMobile/ITMMessenger.swift

```diff
@@ -29,11 +29,29 @@ internal extension JSONSerialization {
             // Return empty JSON string for void.
             return ""
         }
+        let wrapped: Bool
+        let validJSONObject: Any
+        if JSONSerialization.isValidJSONObject(object) {
+            wrapped = false
+            validJSONObject = object
+        } else {
+            wrapped = true
+            // Wrap object in an array
+            validJSONObject = [object]
```

Comment:
Will wrapping an invalid object in an array always make it valid for JSONSerialization? If not, then we should call
```swift
JSONSerialization.isValidJSONObject(validJSONObject)
```

Reply:
This is internal, and as far as I know, it will always work in times that it is called.